src/app/services/markdown-service.service.ts

```diff
@@ -0,0 +1,70 @@
+import { Injectable } from '@angular/core';
```

Comment:
Awesome work with the test spec :)

Reply:
Thanks dood :) I wanna add a few more but i think we're good for now